array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,7 @@
+/* exported initial */
+function initial(array) {
+  array.pop();
```

Comment:
_**NO.**_ This is changing the original array, and that change will persist after the function ends. You need to make your own copy of the array missing that last value without changing the original.